README.md

```diff
@@ -107,17 +107,38 @@ Follow the guidelines in [CONTRIBUTING.md](https://github.com/CSE-25/quick_start
 
 ### Run the package.
 
-1. Run the following command in the terminal to start the package:
+To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the main quick_start_express directory.
```

Comment:
```suggestion
To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the root directory of `quick_start_express`.
```